setup/baseConfig.yaml

```diff
@@ -114,3 +114,16 @@ mountall:
     - <list of containers to be mounted>
   container-denylist:
     - <list of containers not to be mounted>
+
+# health-monitor configuration
+health-monitor:
+  enable-monitoring: true|false <enable health-monitor>
+  blobfuse2-poll-interval: <Blobfuse2 stats polling interval (in sec). Default - 5 sec>
+  stats-poll-interval: <CPU, memory and network usage polling interval (in sec). Default - 10 sec>
```

Comment:
this name sounds incorrect, its not the stats polling rather process level health polling, stats sounds more like the first one. May be "blobfuse2-poll-interval" shall be called "stats-poll-interval" and "stats-poll-interval" shall be "process-moniitor-interval" [Suggestion]

Reply:
Since these values are in sec we should add -sec to the end of the name like for the other Blobfuse options.

Reply:
I feel like the default values should be a little longer, maybe every 60s?

Reply:
I think 60s will be long for both blobfuse stats polling as well as collecting system stats (cpu, mem) for the pid. We can discuss on what should be the ideal default values